.github/workflows/source_heroku.yml

```diff
@@ -40,3 +40,5 @@ jobs:
         run: make gen-docs
       - name: Fail if docs are changed
         run: test "$(git status -s ./docs/tables | wc -l)" -eq 0
```

Comment:
I think this is missing the `gen-code` step

Reply:
Ah, good catch, fixed in https://github.com/cloudquery/cloudquery/pull/2635/commits/4ec26f7e298e5c870e1d68bb66853b4bc178067b